docker-deploy/training_template/docker-compose-spark.yml

```diff
@@ -90,10 +90,11 @@ services:
         set -x
         sleep 5 && python fateflow/python/fate_flow/fate_flow_server.py
     environment:
-      PYTHONPATH: "$PYTHONPATH:/data/projects/fate/fate/python:/data/projects/fate/eggroll/python:/data/projects/fate/fateflow/python:/data/projects/fate/fate/python/fate_client"
+      PYTHONPATH: "$$PYTHONPATH:/data/projects/fate/fate/python:/data/projects/fate/eggroll/python:/data/projects/fate/fateflow/python:/data/projects/fate/fate/python/fate_client"
```

Comment:
Why add a "$"?

Reply:
It can fix the waring of 'WARN[0000] The "PYTHONPATH" variable is not set. Defaulting to a blank string'.

https://docs.docker.com/compose/compose-file/#interpolation